### PR TITLE
Animate charts and tooltip repositioning

### PR DESCRIPTION
**Pull request by gabycampagna**
Change layout for tooltip and charts. Add delay to tooltip (needs more work). New SCSS for styling tooltip.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit ea80f9e7ea763259ac98f7641ce2718c212e70c1

https://deploy-preview-2802--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
Supersedes #2797

**Comment by gabycampagna** [comment]
Tooltips now display on chart. 
screen shot 2017-03-28 at 9 21 54 pm

File: app/components/chart-donut.js [hunk 1]
```diff
@@ -26,26 +27,39 @@ export default Component.extend({
     const width = get(this, 'width');
     const height = get(this, 'height');
     const displayTooltip = get(this, 'displayTooltip');
-    const hideTooltip = get(this, 'hideTooltip');
+    // const hideTooltip = get(this, 'hideTooltip');
```
Comment (jrjohnson) [inline app/components/chart-donut.js]: Please remove commented code.

File: app/components/chart-pie.js [hunk 2]
```diff
@@ -26,37 +27,57 @@ export default Component.extend({
     const width = get(this, 'width');
     const height = get(this, 'height');
     const displayTooltip = get(this, 'displayTooltip');
-    const hideTooltip = get(this, 'hideTooltip');
+    // const hideTooltip = get(this, 'hideTooltip');
```
Comment (jrjohnson) [inline app/components/chart-pie.js]: Please remove commented code.

File: app/components/chart-pie.js [hunk 3]
```diff
@@ -26,37 +27,57 @@ export default Component.extend({
...
     const radius = Math.min(width, height) / 2;
     const color = scaleOrdinal(schemeCategory10);
 
-    let t = transition().duration(2000).ease(easeLinear);
+    let t = transition().duration(500).ease(easeLinear);
 
+    /*let t2 = transition()
```
Comment (jrjohnson) [inline app/components/chart-pie.js]: Please remove commented code.

File: app/components/chart-pie.js [hunk 4]
```diff
@@ -26,37 +27,57 @@ export default Component.extend({
...
+      .attrTween("d", tweenPie);
+
+    function tweenPie(b) {
+      b.innerRadius = 0;
+      var i = interpolate({startAngle: 0, endAngle: 0}, b);
+      return function(t) { return createArc(i(t)); };
+    }
+
+    path.on('mouseover', function(d, index, items) {displayTooltip(d.data, items[index], createLabelArc.centroid(d));});
+    //path.on('mouseout', d => hideTooltip());
```
Comment (jrjohnson) [inline app/components/chart-pie.js]: Please remove commented code.

Comment (gabycampagna) [inline app/components/chart-pie.js]: Commented out code has been removed.  We can wait to merge if you would like.  I am still working on the same branch and do not have the latest updates for master. If I can keep working on my current branch, then I don't see a need to merge.  I'll leave the decision up to you. Thanks!

File: app/components/chart-pie.js (hunk 2 above)
Comment (gabycampagna) [inline app/components/chart-pie.js]: Commented code has been removed. We can wait to merge if you would like to. I will continue working on this branch.

**Comment by gabycampagna** [comment]
Commented code has been removed. We can wait to merge if you would like to. I will continue working on this branch.

**Comment by jrjohnson** [comment]
Ok cool.  Just don't do too much work on the tooltips or layout since I think we're going to change the strategy for those.

**Pull request merged**